src/wallets/EthersWallet.ts

```diff
@@ -60,40 +66,86 @@ export class EthersWallet implements TLWallet, TLSigner {
       password,
       typeof progressCallback === 'function' && progressCallback
     )
+
+    const backup = this.createBackup(encryptedKeystore)
+
     return {
       address: this.address,
-      keystore: encryptedKeystore,
+      backup,
       pubKey: this.pubKey
     }
   }
 
   /**
-   * Encrypts given keystore and loads wallet.
-   * @param encryptedKeystore Encrypted keystore from `createAccount`.
-   * @param password Password to decrypt keystore.
+   * Create a backup following the current backup format from an encryptedKeystore
+   * Can be used to migrate from backup version 0 to backup version 1
+   * @param encryptedKeystore
+   */
+  public createBackup(encryptedKeystore: string): string {
```

Comment:
`serializeWallet` or similar. It should describe that it just creates a serialized version of the wallet and returns that. There is no backup done.